dell/src/test/java/org/apache/iceberg/dell/ecs/TestEcsCatalog.java

```diff
@@ -128,15 +127,13 @@ public void testDropNamespace() {
     ecsCatalog.createNamespace(Namespace.of("a", "b1"));
     ecsCatalog.createTable(TableIdentifier.of("a", "t1"), SCHEMA);
 
-    AssertHelpers.assertThrows(
-        "Drop an unknown namespace should throw exception",
-        NoSuchNamespaceException.class,
-        () -> ecsCatalog.dropNamespace(Namespace.of("unknown")));
+    Assertions.assertThatThrownBy(() -> ecsCatalog.dropNamespace(Namespace.of("unknown")))
+        .isInstanceOf(NoSuchNamespaceException.class)
+        .hasMessageContaining("Namespace %s does not exist", "unknown");
```

Comment:
why not make the namespace name directly part of the expected error msg? `"Namespace unknown does not exist"`
Same for all the other places